indexers/keyvalue/PostgreSQLDBMSIndexer/tests/test_PostgreSQLdbindexer.py

```diff
@@ -0,0 +1,30 @@
+from jina import Document
+from .. import PostgreSQLDBMSIndexer
+
+from tests import get_documents
+
+
+def doc_without_embedding(d):
+    new_doc = Document()
+    new_doc.CopyFrom(d)
+    new_doc.ClearField('embedding')
+    return new_doc
+
+
```

Comment:
Can you also test 

- chunks
- integration test in a flow with an index/update/delete request
- a small benchmark comparison vs BinaryPbIndexer (~10k docs write speed measurements)